batch/batch/driver/main.py

```diff
@@ -679,8 +692,12 @@ async def on_startup(app):
     await scheduler.async_init()
     app['scheduler'] = scheduler
 
-    # asyncio.ensure_future(check_incremental_loop(db))
-    # asyncio.ensure_future(check_resource_aggregation(db))
+    app['check_incremental'] = None
+    app['check_resource_aggregation'] = None
```

Comment:
I think it would be more clear if these were named something like `check_incremental_error` and `check_resource_aggregation_error`